securitas/controller/authentication.py

```diff
@@ -1,43 +1,55 @@
-from flask import flash, redirect, request, session, url_for
+from flask import flash, redirect, session, url_for, render_template
 import python_freeipa
 
 from securitas import app
-from securitas.security.ipa import maybe_ipa_login
-from securitas.utility import with_ipa
+from securitas.form.login_user import LoginUserForm
+from securitas.security.ipa import maybe_ipa_login, maybe_ipa_session
 
 
 @app.route('/logout')
-@with_ipa(app, session)
-def logout(ipa):
+def logout():
+    """Log the user out."""
+    # Don't use the with_ipa() decorator, otherwise anonymous users visiting this endpoint will be
```

Comment:
Heh, fair. I guess I didn't consider that an issue because an anonymous user shouldn't be going to this endpoint. :laughing: